.gitignore

```diff
@@ -3,3 +3,4 @@
 .rvmrc
 Gemfile.lock
 tags
+/.bundle/
```

Comment:
Not _directly_ related to the PR, but after adding `rubocop` to the Gemfile and running `bundle install`, I noticed that the `.bundle` directory wasn't ignored by git.

Reply:
Good call.